authdelegate/Dockerfile

```diff
@@ -3,12 +3,9 @@ FROM dev-standard
 RUN bash -l -c 'go get github.com/18F/authdelegate && \
     strip $GOPATH/bin/authdelegate'
 
-RUN groupadd -r authdelegate && useradd -r -m -g authdelegate authdelegate && \
-    mkdir $APP_SYS_ROOT/authdelegate && \
-    chown authdelegate:authdelegate $APP_SYS_ROOT/authdelegate
```

Comment:
Just out of curiosity, why did these two lines get removed?  It's not readily apparent to me.